tests/models/test_horovod.py

```diff
@@ -71,6 +71,8 @@ def _run_horovod(trainer_options):
     ]
     if trainer_options.get("accelerator", "cpu") == "gpu":
         cmdline += ["--on-gpu"]
+    if devices == 2:
```

Comment:
was it meant to be this?
```suggestion
    if devices > 1:
```

Reply:
Not really, but I suppose we could pass the number of devices instead as well. See also my other comments regarding this option.